### PR TITLE
[DOCS-5738] Tutorial - Enabling Tracing for a Java Application with the Admission Controller

### PR DESCRIPTION
**Pull request by brett0000FF**
### What does this PR do?
Add new tutorial page for enabling tracing for a Java application with the Admission Controller.
kind: guide

### Motivation
Stefano wrote [this detailed tutorial](https://docs.google.com/document/d/1J4bLElHMZVOxc-Hm9OYWPejhbopEpBzGylCqNrGBbxM/edit) that we'd like to publish to our public docs.

### Preview
[New page](https://docs-staging.datadoghq.com/brett0000FF/DOCS-5738/tracing/guide/tutorial-enable-java-admission-controller/)

### Additional Notes
N/A

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Comment by urseberry** [comment]
Docs review on https://datadoghq.atlassian.net/browse/DOCS-5836

File: content/en/tracing/guide/tutorial-enable-java-admission-controller.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+
+{{< code-block lang="shell" >}}
+git clone https://github.com/DataDog/springblog.git
+{{< /code-block >}}
+
+The repository contains a multi-service Java application pre-configured to be run within Docker and Kubernetes. The sample app is a basic Spring app using REST.
+
+## Start and exercise the sample application 
+
+1. Switch to to the `/k8s` directory:
```
Comment (hestonhoffman) [inline content/en/tracing/guide/tutorial-enable-java-admission-controller.md:58]:
```suggestion
1. Switch to to the `/k8s` subdirectory in the springblog repo:
```

File: content/en/tracing/guide/tutorial-enable-java-admission-controller.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+
+1. Install the [Datadog Cluster Agent][5].
+2. Add [Unified Service Tags][6] in pod definition.
+3. [Annotate][7] your pod for library injection.
+4. [Label][8] your pod to instruct the Datadog Admission controller to mutate the pod.
+
+There's no need to add the tracing library because it's automatically injected. You also don't need to add Datadog variables and deploy a new image or version of your app. This section of the tutorial steps you through this process.
+
+1. Install the Datadog Cluster Agent using [this YAML config file](link) and the following command, specifying your own [Datadog API key](/account_management/api-app-keys/):
+   {{< code-block lang="shell" >}}
```
Comment (hestonhoffman) [inline content/en/tracing/guide/tutorial-enable-java-admission-controller.md:102]:
```suggestion
1. From the `k8s` subdirectory, use the following command to install the Datadog Cluster Agent, specifying the `values-with-lib-inj.yaml` config file and your [Datadog API key](/account_management/api-app-keys/):
   {{< code-block lang="shell" >}}
```

File: content/en/tracing/guide/tutorial-enable-java-admission-controller.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+    template:
+        metadata:
+        labels:
+            name: springfront
+            tags.datadoghq.com/env: "dev"
+            tags.datadoghq.com/service: "springfront"
+            tags.datadoghq.com/version: "12"
+        annotations:
+            admission.datadoghq.com/java-lib.version: "latest"
+    {{< /highlight >}}
```
Comment (hestonhoffman) [inline content/en/tracing/guide/tutorial-enable-java-admission-controller.md:164]: Love the highlighting - super cool

File: content/en/tracing/guide/tutorial-enable-java-admission-controller.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+    DD_TRACE_AGENT_URL:  unix:///var/run/datadog/apm.socket
+    URL:                 http://springback:8088
+    JAVA_TOOL_OPTIONS:    -javaagent:/datadog-lib/dd-java-agent.jar
+    Mounts:
+    /datadog-lib from datadog-auto-instrumentation (rw)
+    /var/run/datadog from datadog (rw)
+    /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-qvmtk (ro)
+    ```
+
+8. Verify that the Datadog tracing library is injected into the pod by checking the pod logs:
```
Comment (hestonhoffman) [inline content/en/tracing/guide/tutorial-enable-java-admission-controller.md:223]:
```suggestion
8. Verify that the Datadog tracing library is injected into the pod by checking the pod logs. For example::
```
I suspect the pod name is randomized, so just calling out that this is an example

File: content/en/tracing/guide/tutorial-enable-java-admission-controller.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+    You should see something like this:
+
+    ```
+    Defaulted container "springfront" out of: springfront, datadog-lib-java-init (init)
+    Picked up JAVA_TOOL_OPTIONS:  -javaagent:/datadog-lib/dd-java-agent.jar
+    ```
+
+## View APM traces in Datadog
+
+1. Exercise the app running the following command:
```
Comment (hestonhoffman) [inline content/en/tracing/guide/tutorial-enable-java-admission-controller.md:236]:
```suggestion
1. Run the following command:
```

File: content/en/tracing/guide/tutorial-enable-java-admission-controller.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+   {{< code-block lang="shell" >}}
+curl localhost:8080/upstream{{< /code-block >}}
+
+2. Open the Datadog UI and see the two services reporting under the [Service Catalog][11]:
+   {{< img src="tracing/guide/tutorials/tutorial-admission-controller-service-catalog.png" alt="Springback and springfront services in the Service Catalog." style="width:100%;" >}}
+
+3. Explore Traces and see the associated Service Map as well:
```
Comment (hestonhoffman) [inline content/en/tracing/guide/tutorial-enable-java-admission-controller.md:243]:
```suggestion
3. Explore Traces and see the associated Service Map:
```

**Review by hestonhoffman — COMMENTED** [review]
Great work on this one! Left you some feedback

**Comment by brett0000FF** [comment]
> Great work on this one! Left you some feedback

@hestonhoffman - Thank you for the great edits! I applied all of your suggestions.

**Review by hestonhoffman — APPROVED** [review]
LGTM!

**Pull request merged**